### PR TITLE
Player

### PR DESCRIPTION
**Pull request by EranDaniel98**

### CODE REVIEW

File: Player.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+{
+    enum PieceColor { Black, White, Green };
+
+    class Player
+    {
+        public enum PlayerNumber { FirstPlayer, SecondPlayer, ThirdPlayer }
+        public static Dictionary<PlayerNumber, PieceColor> PieceColorByPlayer = new Dictionary<PlayerNumber, PieceColor>();
+
+        public string nickname;
+        public int number_Of_Wins;
```
Comment (NitsanLitov) [inline Player.cs:14]: c# conventions doesnt use underscore for variables

File: Player.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+        public Player(string nickname)
+        {
+            this.nickname = nickname;
+            this.number_Of_Wins = 0;
+            this.number_Of_loses = 0;
+            InitPlayerColor();
+        }
+
+        private void InitPlayerColor()
```
Comment (NitsanLitov) [inline Player.cs:26]: This is static so why should this happen for every instance?

File: Player.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections.Generic;
+
+namespace SuperChess
+{
+    enum PieceColor { Black, White, Green };
+
+    class Player
+    {
+        public enum PlayerNumber { FirstPlayer, SecondPlayer, ThirdPlayer }
```
Comment (NitsanLitov) [inline Player.cs:10]: why is this part of the class?

File: Player.cs [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        public static Dictionary<PlayerNumber, PieceColor> PieceColorByPlayer = new Dictionary<PlayerNumber, PieceColor>();
+
+        public string nickname;
```
Comment (NitsanLitov) [inline Player.cs:13]: use properties

File: Player.cs [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+using System.Collections.Generic;
+
+namespace SuperChess
+{
+    enum PieceColor { Black, White, Green };
+
+    class Player
+    {
+        public enum PlayerNumber { FirstPlayer, SecondPlayer, ThirdPlayer }
+        public static Dictionary<PlayerNumber, PieceColor> PieceColorByPlayer = new Dictionary<PlayerNumber, PieceColor>();
```
Comment (NitsanLitov) [inline Player.cs:11]: This is static so why should this happen for every instance?

File: Player.cs [hunk 6]
```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections.Generic;
+
+namespace SuperChess
+{
+    enum PieceColor { Black, White, Green };
+
+    class Player
```
Comment (NitsanLitov) [inline Player.cs:8]: why does the player doesn't have a color? how can we know what pieces to play when its his turn?

File: Player.cs [hunk 7]
```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections.Generic;
+
+namespace SuperChess
```
Comment (NitsanLitov) [inline Player.cs:4]: should this really be the same namespace?

File: Player.cs [hunk 8]
```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Collections.Generic;
+
+namespace Players
```
Comment (NitsanLitov) [inline Player.cs:4]: why players? and not player?

File: Player.cs (hunk 4 above)
Comment (EranDaniel98) [reply Player.cs]: Did it, Done it :)

File: Player.cs (hunk 3 above)
Comment (EranDaniel98) [reply Player.cs]: not anymore!!

File: Player.cs (hunk 2 above)
Comment (EranDaniel98) [reply Player.cs]: You are right !! I've changed it

File: Player.cs (hunk 1 above)
Comment (EranDaniel98) [reply Player.cs]: why not :(

File: Player.cs [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+{
+    enum PieceColor { White, Black, Green };
+    enum PlayerNumber { FirstPlayer, SecondPlayer, ThirdPlayer };
+
+    static class PlayerColor
+    {
+        public static Dictionary<PlayerNumber, PieceColor> PieceColorByPlayer = new Dictionary<PlayerNumber, PieceColor>()
```
Comment (NitsanLitov) [inline Player.cs:11]: private

Comment (EranDaniel98) [reply Player.cs]: You are right ! Done

**Review by NitsanLitov — APPROVED** [review]

**Pull request merged**